tools/js_optimizer.py

```diff
@@ -543,8 +543,13 @@ def write_chunk(chunk, i):
 
       if 'last' in passes and len(funcs):
         count = funcs[0][1].count('\n')
-        if count > 3000:
-          print('warning: Output contains some very large functions (%s lines in %s), consider building source files with -Os or -Oz, and/or trying OUTLINING_LIMIT to break them up (see settings.js; note that the parameter there affects AST nodes, while we measure lines here, so the two may not match up)' % (count, funcs[0][0]), file=sys.stderr)
+        if count > 3000 and not shared.Building.is_wasm_only:
```

Comment:
`is_wasm_only` is a function, so need `()` so that we call it (otherwise it always evaluates to true).

Reply:
Thanks for the review! Definitely -- thought it was a property for some reason, will fix it.